src/components/structures/MessagePanel.tsx

```diff
@@ -712,6 +712,8 @@ export default class MessagePanel extends React.Component<IProps, IState> {
 
         const callEventGrouper = this.callEventGroupers.get(mxEv.getContent().call_id);
 
+        const isDirectMessage = this.props.room?.getJoinedMemberCount() <= 2;
```

Comment:
This doesn't do the same thing as `this.props.room?.getMembers().length <= 2` the latter of which includes not joined members (invited, left, etc)

What is the target functionality here?

Reply:
If a room has 2 members then we do not need to display the sender as we won't need to disambiguate the name, nor display the mixd.

Reply:
Okay but in a room of 3 people, if one leaves, now all the history will be impossible to tell who is who